br/backup-and-restore-overview.md

```diff
@@ -1,114 +1,135 @@
+### Some tips
 
-#### Version check
+Snapshot backup:
 
-Before performing backup and restoration, BR compares and checks the TiDB cluster version and the BR version. If there is a major-version mismatch (for example, BR v4.x and TiDB v5.x), BR prompts a reminder to exit. To forcibly skip the version check, you can set `--check-requirements=false`. Note that skipping the version check might introduce incompatibility.
+- It is recommended that you perform the backup operation during off-peak hours to minimize the impact on applications.
+- It is recommended that you execute multiple backup or restore operations one by one. Running backup operations in parallel reduces performance and also affects online applications. Worse still, lack of collaboration between multiple tasks might result in task failures and affect cluster performance.
 
-The version compatibility mapping between BR and TiDB versions are as follows:
+Snapshot restore:
 
-| Backup version (vertical) \ Restoration version (horizontal) | Use BR v5.4 to restore TiDB v5.4 | Use BR v6.0 to restore TiDB v6.0 | Use BR v6.1 to restore TiDB v6.1 | Use BR v6.2 to restore TiDB v6.2 |
-|  ----  |  ----  | ---- | ---- | ---- |
-| Use BR v5.4 to back up TiDB v5.4 | Compatible | Incompatible (Before restoration, modify the restoration cluster to use the same [new collation](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) as the backup cluster.) | Incompatible (Before restoration, modify the restoration cluster to use the same [new collation](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) as the backup cluster.) | Incompatible (Before restoration, modify the restoration cluster to use the same [new collation](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) as the backup cluster.) |
-| Use BR v6.0 to back up TiDB v6.0 | Incompatible | Compatible | Compatible | Compatible |
-| Use BR v6.1 to back up TiDB v6.1 | Incompatible | Compatible (A known issue [#36379](https://github.com/pingcap/tidb/issues/36379): if backup data contains an empty schema, BR might report an error.) | Compatible | Compatible |
-| Use BR v6.2 to back up TiDB v6.2 | Incompatible | Compatible (A known issue [#36379](https://github.com/pingcap/tidb/issues/36379): if backup data contains an empty schema, BR might report an error.) | Compatible | Compatible |
+- BR uses resources of the target cluster as much as possible. Therefore, it is recommended that you restore data to a new cluster or an offline cluster. Avoid restoring data to a production cluster. Otherwise, your application might be affected. PITR only supports restoring data to an empty cluster.
```

Comment:
```suggestion
- BR uses resources of the target cluster as much as possible. Therefore, it is recommended that you restore data to a new cluster or an offline cluster. Avoid restoring data to a production cluster. Otherwise, your application might be impacted. PITR only supports restoring data to an empty cluster.
```